en/payloads/README.md

```diff
@@ -4,24 +4,37 @@ PX4 supports a wide range of payloads and cameras.
 
 ## Mapping Drones
 
-* [Camera triggering](../peripherals/camera.md) via GPIO out
-* [Camera triggering](../peripherals/camera.md) via PWM out
-* [Camera triggering](../peripherals/camera.md) via MAVLink out
-* [Camera timing](../peripherals/camera.md#camera_capture) feedback via hotshoe input
+Mapping drones use cameras to capture images at time or distance intervals during surveys.
 
-## Cargo drones and alike: Servos / Outputs
+The following topics show how to connect your camera and set up camera triggering:
+* [Camera triggering](../peripherals/camera.md) from flight controller PWM or GPIO outputs, or via MAVLink.
+* [Camera timing feedback](../peripherals/camera.md#camera_capture) via hotshoe input.
 
-* Servo or GPIO triggering (via RC or via commands)
+The MAVLink commands used in survey missions (and as standalone commands) are:
+* [MAV_CMD_DO_SET_CAM_TRIGG_INTERVAL](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_CAM_TRIGG_INTERVAL) - set time interval between captures.
```

Comment:
@julianoes These are the three messages used by the camera trigger driver (I will need to check that). For a mavlink camera these just get forwarded by PX4, which is how triggering works on a mavlink camera.

Can you confirm these are what are used by the MAVSDK camera API? Specifically I suspect you use these for the following:
[Camera.take_photo_async()](https://mavsdk.mavlink.io/develop/en/api_reference/classmavsdk_1_1_camera.html#classmavsdk_1_1_camera_1a343217bf38cc71cd258c7e81626dccb5)
take_photo()
start_photo_interval_async()
start_photo_interval()
stop_photo_interval_async()
stop_photo_interval()
If so, good for me to mention this API rather than/as well as the messages. So wanted to check.

Reply:
Yes, these are the functions but not that they are not compatible with the trigger command above.

It's a bit tricky, we basically have triggering, and we have direct "take picture" camera commands, and these are not the same thing.

Reply:
Thanks @julianoes. That's a bit crap because the _TRIG commands work on FC and probably most MAVLink connected cameras, while the camera protocol works on a small subset.

- Do any cameras support the protocol or are we using an interface layer?
- Might be nice to have feature detection and use the _TRIG if the camera protocol isn't supporte.